src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/VS97AddItemCommandHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    [ExportCommandGroup(CommandGroup.VisualStudioStandard97)]
+    [AppliesTo(ProjectCapability.DotNet)]
+    internal class VS97AddItemCommandHandler : AbstractAddItemCommandHandler
+    {
+        protected override Dictionary<long, List<TemplateDetails>> GetTemplateDetails() => new Dictionary<long, List<TemplateDetails>>
+        {
+           // Command Id                                Capability                      DirNamePackageGuid          DirNameResourceId                                       TemplateNameResourceId
```

Comment:
TemplateNamePackageGuid

Reply:
No, this is actually TemplateNameResourceId. This is another convenience overload for when both resources come from the same package, but clearly its more inconvenient than it appears so I'll just specify things twice :)